proctor-consumer/src/test/java/com/indeed/proctor/consumer/spring/TestProctorConsumerUtils.java

```diff
@@ -228,15 +228,11 @@ public void testParseForceGroupsList() {
         assertThat(parseForceGroupsList("fasdfasdf")).isEmpty();
         assertThat(parseForceGroupsList("test" + Integer.MAX_VALUE + "0")).isEmpty();
         assertThat(parseForceGroupsList("test-")).isEmpty();
-        assertThat(parseForceGroupsList("test0-")).isEmpty();
```

Comment:
Should this behavior be maintained where a force group is ignored if extra text is in the url after or can it be ignored?

Reply:
I like to have the syntax strict, so I would keep it.

Reply:
Reverted